lib/matplotlib/backend_bases.py

```diff
@@ -1238,9 +1237,9 @@ class ResizeEvent(Event):
 
     Attributes
     ----------
-    width : scalar
+    width : float
```

Comment:
same as above, these may well be ints.

Reply:
They actually are: `self.width, self.height = canvas.get_width_height()` with all implementations of the function returning ints.